bids/grabbids/config/bids.json

```diff
@@ -41,6 +41,10 @@
             "name": "run",
             "pattern": "[_/\\\\]run-0*(\\d+)"
         },
+        {
+            "name": "acq",
```

Comment:
This is a bit problematic, as it will create two separate entities `acq` and `acquisition` that always have the same value. It's not the end of the world, but is ugly and inefficient. @effigies opened a PR to support aliasing that I think I've dropped the ball on several times now (though I think it may no longer work following changes to grabbit--need to look into that). Not sure what to do about this right now; mostly just flagging this for consideration.

Reply:
I'll rebase. I don't think it's ever been working, but maybe now it will...

Reply:
Yeah if I remember correctly, aliasing used to work (it always confused me actually)

Reply:
okay, so I believe this was a mistake on my end. I didn't notice there was already an entity called `acquisition` ...